packages/auth-middleware/.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
Is separate prettier config required? We have one already in the root of the monorepo

Reply:
Yeah, I added that by mistake.